### ISSUE TITLE
Download Chromium should support ARM64 Win32

### ISSUE THREAD

**Issue by randomascii** [beaufortfrancois/download-chromium#50]
Chromium for Windows now supports a new target platform - ARM64. This is a native version of Chromium (not a store app or UWP app or universal app) that runs on the (relatively) new ARM64 version of Windows. There is an FYI build machine that currently builds all targets for release:

https://ci.chromium.org/p/chromium/builders/luci.chromium.ci/win32-arm64-rel

Currently the build has been green for 98 of the last 100 builds.

It would be nice to have an easy way to download Chromium builds for this platform.

There is a separate Chromium bug for this request (crbug.com/940303) which then links to the master Chromium ARM64 Win32 bug (crbug.com/893460).

**Comment by beaufortfrancois** [comment]
@randomascii Can you point to me which folder in https://storage.googleapis.com/chromium-browser-snapshots/index.html is the one containing ARM64 Win32 builds?

**Comment by randomascii** [comment]
Good question. I'm not sure that the builds are going there. Do you know how that is configured?

The most recent FYI build of ARM64 Win32 is at https://ci.chromium.org/p/chromium/builders/ci/win32-arm64-rel/1255 and was for revision 644789. I don't see a folder on the snapshots page that looks like it would hold ARM64 Win32 builds, so I assume that there is another step which needs to be done first.

Do you know to get that storage area populated?

**Comment by beaufortfrancois** [comment]
I don't sadly.
Maybe @paulirish has some pointers.

**Comment by louis993546** [comment]
is this the folder?
https://storage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Win_Arm64/
